backend/controller/sql/queries.sql

```diff
@@ -645,6 +645,20 @@ UPDATE SET
   topic_id = excluded.topic_id
 RETURNING id;
 
+-- name: DeleteOldSubscriptions :exec
```

Comment:
nitty nitty naming nit: should this be `DeleteUnusedSubscriptions` or `DeleteObsoleteSubscriptions` since the selection criteria is whether the subscription is used, not whether it's aged past a certain TTL (time to live) date?

Reply:
That's better naming 👍

Reply:
(Renamed again because its changed whats it's doing)